README.md

```diff
@@ -3,17 +3,20 @@
 
 This is a simple **\*nix** script for installing all the essentials for a simple Spigot server. It downloads the latest version of:
 
-* Spigot
-* EssentialsX
-* ProtocolLib
-* Multiverse (Core and NetherPortals)
-* PlugMan
-* PermissionsEx
-* Vault
-* WorldEdit
-* WorldGuard
-
-It also creates a simple start script that runs your server with *4GB of RAM*, and automatically sets the EULA to agreed. (*Note: by using this script __you are agreeing to the Mojang EULA__*)
+* Spigot(http://spigotmc.org)
```

Comment:
Please use proper markdown link format, e.g. `[text](url)`

Reply:
(Not gonna lie you were the one to put those there but sure okay)